transform/mattermost-analytics/models/intermediate/product/features/mattermost/int_feature_daily_usage_per_user.sql

```diff
@@ -1,35 +1,57 @@
--- Forcing dependency due to load_feature_mappings macro
--- depends_on: {{ ref('tracking_plan') }}
-
 {{
     config({
-        "materialized": "table",
+        "materialized": "incremental",
```

Comment:
@ifoukarakis , forgot to mention - are there any new client event names for the new features  to be added to the `tracking_plan`?

Reply:
No. All new events are from server side. This includes read only channels and scheduled posts. features A few events are also available for guest accounts, which also exist on client-side. 

I'll create update confluence, documenting the known values used in feature names.

Reply:
Thanks, documenting the feature events would be really helpful. I'm going through the list in https://docs.google.com/document/d/1u7UcVv6zaFIsR4YzhlJptZZSHv0M6ERqv9jPSGtp8po/edit?tab=t.0 - to identify the events that are not yet captured in code, so having a reference list (other than scouring PRs, tracking list or querying the RAW data) would help.